desktop/core/src/desktop/lib/botserver/views.py

```diff
@@ -15,11 +15,69 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import logging
+import json
+from pprint import pprint
+
 from django.shortcuts import render
 from django.http import HttpResponse
-from desktop.lib.django_util import login_notrequired
+from desktop.lib.django_util import login_notrequired, JsonResponse
+from django.views.decorators.csrf import csrf_exempt
+
+from django.conf import settings
+from slack_sdk import WebClient
+
+LOG = logging.getLogger(__name__)
+
+SLACK_VERIFICATION_TOKEN = getattr(settings, 'SLACK_VERIFICATION_TOKEN', None)
```

Comment:
Should look more like SLACK_VERIFICATION_TOKEN.get()

Reply:
Still there..

Reply:
In settings.py, desktop.conf.SLACK.SLACK_VERIFICATION_TOKEN.get() is used and from settings.py, we are getting the token here

Do I need to directly get() the token in views.py from desktop.conf and removing settings.py from between? @romainr

Reply:
Please remove them from desktop/settings.py and just use SLACK.SLACK_VERIFICATION_TOKEN.get() here

Reply:
Fixed